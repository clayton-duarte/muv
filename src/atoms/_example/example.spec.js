import React from 'react';

import Component from './index';

it('should render a component', () => {
  const wrapper = shallow(
    <Component />,
  ).dive();
  expect(wrapper.props().color).toEqual('tomato');
  expect(wrapper).toMatchSnapshot();
});

it('should render a component with a prop', () => {
  const wrapper = shallow(
    <Component strong />,
  ).dive();
  expect(wrapper).toMatchSnapshot();
});

it('should render a component with a color', () => {
  const wrapper = shallow(
    <Component color="black" />,
  ).dive();
  expect(wrapper).toMatchSnapshot();
});
