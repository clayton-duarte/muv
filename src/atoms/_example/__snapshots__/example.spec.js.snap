// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a component 1`] = `
<p
  className="sc-bdVaJa EuHED"
  color="tomato"
>
  Default text
</p>
`;

exports[`should render a component with a prop 1`] = `
<p
  className="sc-bdVaJa jbrfPx"
  color="tomato"
>
  Default text
</p>
`;
